app/serializers/site_serializer.rb

```diff
@@ -163,4 +162,16 @@ def include_shared_drafts_category_id?
     scope.can_create_shared_draft?
   end
 
+  private
+
+  def ordered_flags(flags)
+    notify_moderators_type = 7
```

Comment:
Isn't this constant available elsewhere?

Reply:
It's not. I'll make it a constant 👍

Reply:
I realized I should use `PostActionType.flag_types[:notify_moderators]` instead.